packages/ckeditor5-engine/src/view/domconverter.ts

```diff
@@ -558,7 +559,8 @@ export default class DomConverter {
 			}
 
 			const transparentRendering = childView.is( 'element' ) &&
-				( childView.getCustomProperty( 'dataPipeline:transparentRendering' ) as boolean | undefined );
+				( childView.getCustomProperty( 'dataPipeline:transparentRendering' ) as boolean | undefined ) &&
```

Comment:
```suggestion
				!!childView.getCustomProperty( 'dataPipeline:transparentRendering' ) &&
```

Reply:
Or leave it `unknown`. It's used in boolean context only - it doesn't matter what's inside.